charts/delphai-keda/templates/ingress.yml

```diff
@@ -3,41 +3,36 @@
   {{ $domains = uniq (concat $domains (list "delphai.live" "delphai.com")) }}
 {{ end }}
 
-{{ range $domain_index, $domain := $domains }}
-
-{{ if and $.Values.http.enabled $.Values.http.public }}
-{{ if not (lookup "v1" "Service" "ambassador" "ambassador") }}
+{{ if or (and $.Values.http.enabled $.Values.http.public) (and $.Values.grpc.enabled $.Values.grpc.public) }}
 ---
 apiVersion: getambassador.io/v2
 kind: Host
 metadata:
-  name: {{ $.Release.Name }}-domain{{ $domain_index }}-http
+  name: {{ $.Release.Name }}
   namespace: {{ $.Release.Namespace }}
-  annotations:
-    {{ if (lookup "v1" "Service" "emissary" "emissary-ingress") }}
-    external-dns.ambassador-service: emissary/emissary-ingress
-    {{ else if (lookup "v1" "Service" "ambassador" "ambassador") }}
-    external-dns.ambassador-service: ambassador/ambassador
-    {{ end }}
 spec:
-  hostname: "{{ $.Values.subdomain }}.{{ $domain }}"
+  hostname: "*"
```

Comment:
I've changed that we'll have exactly one host even if we'll serve more TLDs simultaneously. In any case final descision is made by Mapping. Host does almost nothing and isn't well supported by emissary — so lets "ignore" it